src/topology/category/TopCommRing.lean

```diff
@@ -20,11 +20,12 @@ structure TopCommRing :=
 
 namespace TopCommRing
 
+attribute [instance] is_comm_ring is_topological_space is_topological_ring
```

Comment:
Idem dito, do these trigger for coerced TopCommRings?

Reply:
```suggestion
— TODO revisit this after #1438 merges, to check coercions and instances are handled consistently
attribute [instance] is_comm_ring is_topological_space is_topological_ring
```